x/ecocredit/server/core/retire.go

```diff
@@ -2,6 +2,7 @@ package core
 
 import (
 	"context"
+	"github.com/regen-network/regen-ledger/x/ecocredit"
```

Comment:
some of these imports didn't appear to join the existing /regen-ledger/ group

Reply:
I just updated my golang preferences too. Thanks for sharing that over at the watercooler. :slightly_smiling_face: